package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@codewell/is-defined",
-  "version": "1.0.4",
+  "version": "1.1.0",
```

Comment:
technically this is a major release right, its not compat with 1.x?

Reply:
Yea, I was thinking about which is the correct one and I think you are right.